### PR TITLE
Azure hubs doc reference updated (Rich App)

### PR DESCRIPTION
**Pull request by JV0812**
## Purpose of this pull request

This pull request... 

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: docs/integrations/microsoft-azure/active-directory-azure.md [hunk 1]
```diff
@@ -21,21 +21,20 @@ Only global administrators, security administrators, security readers, and repor
 
 The Sumo Logic app for Azure Active Directory presents information about activity in Azure Active Directory, including role management, user management, group management, directory management, and application management.
 
+## Prerequisites
+
+* An Azure subscription must be associated (attached) to AAD. For more information, see this [Azure Active Directory documentation](https://docs.microsoft.com/en-us/azure/active-directory/fundamentals/active-directory-how-subscriptions-associated-directory).
```
Comment (jpipkin1) [inline docs/integrations/microsoft-azure/active-directory-azure.md:26]:
```suggestion
* An Azure subscription must be associated (attached) to AAD. For more information, see [Azure Active Directory documentation](https://docs.microsoft.com/en-us/azure/active-directory/fundamentals/active-directory-how-subscriptions-associated-directory).
```

**Review by jpipkin1 — APPROVED** [review]

File: docs/integrations/microsoft-azure/active-directory-azure.md [hunk 2]
```diff
@@ -21,21 +21,20 @@ Only global administrators, security administrators, security readers, and repor
...
+* An Azure subscription must be associated (attached) to AAD. For more information, see the [Azure Active Directory documentation](https://docs.microsoft.com/en-us/azure/active-directory/fundamentals/active-directory-how-subscriptions-associated-directory).
+* To export Azure Activity logs to reports, be sure you have met the [Azure Active Directory requirements](https://docs.microsoft.com/en-us/azure/active-directory/reports-monitoring/concept-activity-logs-azure-monitor).
+
 ## Collect logs for the Azure Active Directory app
 
 Sumo Logic supports several methods for collecting logs from Event Hub. You can choose any of them to collect logs.
 
-- [Azure Event Hubs Source](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/azure-event-hubs-source/) (Recommended)
+- [Azure Event Hubs Source](//docs/send-data/collect-from-other-data-sources/azure-monitoring/ms-azure-event-hubs-source/#setup) (Recommended)
 - Perform Steps 1 and Step 2 of [Collect Logs from Azure Monitor using Azure Functions](/docs/send-data/collect-from-other-data-sources/azure-monitoring/collect-logs-azure-monitor/#configure-log-collection)
```
Comment (himanshu219) [inline docs/integrations/microsoft-azure/active-directory-azure.md:34]: as mentioned in the jira, we should remove this whole section 
```
Sumo Logic supports several methods for collecting logs from Event Hub. You can choose any of them to collect logs.

- [Azure Event Hubs Source](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/azure-event-hubs-source/) (Recommended)
- Perform Steps 1 and Step 2 of [Collect Logs from Azure Monitor using Azure Functions](/docs/send-data/collect-from-other-data-sources/azure-monitoring/collect-logs-azure-monitor/#configure-log-collection)
```
Instead replace it with 
`To set up the logs collection in Sumo Logic portal, refer to our [Azure Event Hubs for Logs documentation]`

Same comment for all the rest of the pages

Comment (jpipkin1) [reply docs/integrations/microsoft-azure/active-directory-azure.md]: @himanshu219, I made the changes with commit "[Updates per review by Himansu Pal](https://github.com/SumoLogic/sumologic-documentation/pull/4032/commits/146f096aa919a5d2868c3c4466bac479e97d5ade)".

**Comment by jpipkin1** [comment]
@himanshu219, I made your requested changes. Could you review it again?

**Review by himanshu219 — APPROVED** [review]

**Pull request merged**